recipe/patches/0001-Disable-unity-build-for-permission-plugins-in-macos.patch

```diff
@@ -0,0 +1,70 @@
+From 475344bfd9b92d2bc45291a3f8c8dfb3d0e88470 Mon Sep 17 00:00:00 2001
+From: Rafael Martins <rmartins@anaconda.com>
+Date: Thu, 20 Jun 2024 00:52:32 +0200
+Subject: [PATCH 1/2] Disable unity build for permission plugins in macos
```

Comment:
```suggestion
Subject: [PATCH 1/2] Disable unity build for permission plugins in macos

Unity builds were leading to issues because of conflicts between symbols. We disable this on a per-plugin basis rather than globally as that would lead to very long build times.
```